mlflow/mleap.py

```diff
@@ -1,8 +1,11 @@
 """
 The ``mlflow.mleap`` module provides an API for saving Spark MLLib models using the
 `MLeap <https://github.com/combust/mleap>`_ persistence mechanism.
-A companion module for loading MLflow models with the MLeap flavor format is available in the
-``mlflow/java`` package.
+
+NOTE:
+
+    You cannot load the MLeap model flavor in Python; you must load it using the
+    Java API method ``downloadArtifacts(String runId￼)``.
```

Comment:
@stbof `downloadArtifacts()` will download the model; it will not load it. `MLeapLoader.loadPipeline()` can be used to load the model (https://github.com/mlflow/mlflow/blob/0d42369f2badfbb9ee8cbfbabfc8edf27883d9a2/mlflow/java/scoring/src/main/java/org/mlflow/mleap/MLeapLoader.java#L13)